### ISSUE TITLE
Reduced INFO prints

### ISSUE THREAD

**Issue by GunnyWaffle** [replaceitem/carpet-discarpet#22]
This is a quality of life request.
Is it possible to have a reduced INFO option in the config? Where the following is not printed to server console/logs every 15min or so?
```
[12:17:45] [ReadingThread/INFO]: Websocket closed with reason 'Discord commanded a reconnect (Received opcode 7)' and code COMMANDED_RECONNECT (4999) by client!
[12:17:45] [ReadingThread/INFO]: Trying to reconnect/resume in 1 seconds!
[13:17:21] [WritingThread/INFO]: Websocket closed with reason 'No more WebSocket frame from the server.' and code UNCONFORMED (1002) by client!
[13:17:21] [WritingThread/INFO]: Trying to reconnect/resume in 1 seconds!
```

I'm not certain if the second print is from Discarpet or Scarpet-Additions.

Thank you for considering!

**Comment by replaceitem** [comment]
Thats a thing i wanted to do since i saw those messages myself, and i already tried that two times, failing both times. The problem is that (i think) Javacord somehow wraps the Log4j logger, so the normal ways of disabling Log4j messages did not work sadly. I will see if i get that to work one day though.

**Comment by GunnyWaffle** [comment]
Well that is a bummer. I hope you find a solution some day, best of luck!

I did a quick dig myself, that fallback logger in javacord needs more exposure in their API it seems. You can only set DEBUG/TRACE levels. That's a shame. :/

**Comment by replaceitem** [comment]
Fixed in b7a2a331faab530e2d433e44f5918dfebf7833db 🥳
